Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM mesosphere/mesos:0.21.1-1.1.ubuntu1404
+
+ENV HDFS_MESOS_VERSION 0.1.0
+
+COPY hdfs-mesos-$HDFS_MESOS_VERSION.tgz /hdfs/
+RUN cd /hdfs && tar xfz hdfs-mesos-$HDFS_MESOS_VERSION.tgz
+
+ENV MESOS_NATIVE_JAVA_LIBRARY /usr/local/lib/libmesos.so
+ENV MESOS_NATIVE_LIBRARY /usr/local/lib/libmesos.so
+
+WORKDIR /hdfs/hdfs-mesos-$HDFS_MESOS_VERSION
+
+CMD ./bin/hdfs-mesos
```

Comment:
Can we either modify this to be the entrypoint or change `/bin/hdfs-mesos` to accept `$JAVA_OPTS` to pass to java? That way we can override hadoop configuration with system properties.